src/pages/promocodes/promocode-list-page.js

```diff
@@ -56,6 +56,7 @@ const fieldNames = [
   },
   { columnKey: "tags", value: "tags" },
   { columnKey: "owner_email", value: "owner_email" },
+  { columnKey: "owner", value: "owner" },
```

Comment:
this column is new and should be following the same logic as owner_email
please do check screen cap, bc its break all grid layout
<img width="1503" alt="Screenshot 2024-10-24 at 16 43 08" src="https://github.com/user-attachments/assets/5f7498ed-f0ce-41a8-84df-5933261a0434">